mlflow/langchain/__init__.py

```diff
@@ -1020,6 +1021,29 @@ def autolog(
             MlflowLangchainTracer as a callback during inference. If ``False``, no traces are
             collected during inference. Default to ``True``.
     """
+    enabled_configs = {
+        key
+        for key, value in {
+            "log_input_examples": log_input_examples,
+            "log_model_signatures": log_model_signatures,
+            "log_models": log_models,
+            "log_datasets": log_datasets,
+            "registered_model_name": registered_model_name,
+            "extra_tags": extra_tags,
+        }.items()
+        if value is True
+    }
+    if any(enabled_configs):
+        color_warning(
+            "The follow parameters are deprecated in langchain autologging and will be removed "
+            f"in a future release: `{', '.join(enabled_configs)}`. Langchain autologging no longer "
+            "support automatic model logging and its related attributes. Please log your model "
```

Comment:
`no longer supports` sounds like `is already unsupported`. Can we adjust the exrepssion?